src/elements/content-sidebar/activity-feed/common/activity-message/TruncatableMessage.js

```diff
@@ -0,0 +1,39 @@
+// @flow
+import * as React from 'react';
+import classNames from 'classnames';
+import ToggleMoreLessButton from './ToggleMoreLessButton';
+import './ActivityMessage.scss';
+
+type Props = {
+    children: React.Node,
+};
+
+export default function TruncatableMessage({ children }: Props) {
+    const [isTruncated, setTruncated] = React.useState(true);
+    const [shouldTruncate, setShouldTrucate] = React.useState(false);
```

Comment:
I think this is being renamed but in case not:
```suggestion
    const [shouldTruncate, setShouldTruncate] = React.useState(false);
```